thirdparty/cnpy/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.0 FATAL_ERROR)
+if(COMMAND cmake_policy)
+	cmake_policy(SET CMP0003 NEW)
+endif(COMMAND cmake_policy)
+
+project(CNPY)
+
+set(BUILD_SHARED_LIBS OFF)  
```

Comment:
`BUILD_SHARED_LIBS` is useless because later in this code we explicitly build `cnpy` as static library

Reply:
It was deleted.